tests/subsys/zigbee/osif/serial/serial_async_api/testcase.yaml

```diff
@@ -4,5 +4,5 @@ tests:
     tags: osif_serial
     harness_config:
       fixture: shorted_uart_1
-    integration_platforms:
-      - null # No integration platforms because test requires shorted UART 1 pins
+    # integration_platforms:
+    #  - null # No integration platforms because test requires shorted UART 1 pins
```

Comment:
Before merging this PR, then those lines should either be completely removed or adjusted to some proper code.

Comment placed here to not accidentally merge this PR with commented lines.

Reply:
what is the status on this one, @PerMac ?

Reply:
@tejlmand Do you consider such commented lines as an error? I've done it by purpose as a temporary solution, until this is resolved in the upstream https://github.com/zephyrproject-rtos/zephyr/pull/43451
I want all our samples/tests/applications to have `integration_platforms` for reliability of the CI. However, there is no automated check for this yet, so we can leave with those lines just being deleted for now, and added back when the linked PR is resolved in the upstream

Reply:
has been removed, marking as resolved, @tejlmand  please re-review

Reply:
> Do you consider such commented lines as an error?

yes, cause users of NCS have no idea why such lines are commented out.
Is it wip that has been abandoned, is it because something is not working (a unresolved bug ?), or similar.
And we leave the user clueless, so better to avoid this.

Also their is a risk of such commented code to be forgotten (or left for long time), simply because priorities changes.

So several good reasons to just remove such code completely, and then re-add (or simply revert the commit that removed those lines). You can then easily refer to such commit SHA in the corresponding jira task.

> I want all our samples/tests/applications to have integration_platforms for reliability of the CI.  However, there is no automated check for this yet

Sounds like a very good thing, but I then hope commented out code is not treated as `integration_platforms` being present 😉